### PR TITLE
Adjust/remove obsolete links

### PR DESCRIPTION
**Pull request by Godin**
I've got an email :
> Hello Evgeny,
>
> I'm not certain you're the best person to contact, but yours is the first email address I can find, so hopefully you can pass it on as appropriate.
>
> I installed EclEmma recently to do code coverage analysis of our test suite. In the resources section of the EclEmma website, the following at least are dead links:
>
>Tony Obermeit: "Code Coverage Lessons" - Read why Tony recommends 100% code coverage for your test suites.
> Andrew Glover: "Don't be fooled by the coverage report" - This article shows some risks when misusing coverage reports in projects.
>
> Best wishes,
>
> -- 
> Frederic Heath-Renn
> Twickets, Ltd.

Response was - to use mailing list in future for such notifications.
But anyway we should do a check followed by a fix.

### CODE REVIEW

**Comment by Godin** [comment]
@marchof could you please review?

File: content/resources.html [hunk 1]
```diff
@@ -55,11 +53,7 @@ <h1 style="clear:left">Resources</h1>
 <h2>Articles About EclEmma</h2>
 
 <ul>
-  <li>Tests f&uuml;r die Tests by Marc R. Hoffmann, <a class="extern" href="http://it-republik.de/jaxenter/eclipse-magazin-ausgaben/Android-000442.html">Eclipse Magazin Vol. 3.2011</a>, 2011/03/25 [de]</li>
```
Comment (marchof) [inline content/resources.html]: As these paper publications still exist I wouldn't remove them from the list. Just the web links should be removed.

Comment (Godin) [inline content/resources.html]: @marchof makes sense, fixed

File: content/resources.html [hunk 2]
```diff
@@ -55,11 +53,10 @@ <h1 style="clear:left">Resources</h1>
 <h2>Articles About EclEmma</h2>
 
 <ul>
-  <li>Tests f&uuml;r die Tests by Marc R. Hoffmann, <a class="extern" href="http://it-republik.de/jaxenter/eclipse-magazin-ausgaben/Android-000442.html">Eclipse Magazin Vol. 3.2011</a>, 2011/03/25 [de]</li>
-  <li>EclEmma by Richard Attermeyer, <a class="extern" href="http://it-republik.de/jaxenter/eclipse-magazin-ausgaben/Plug-in-Parade-000340.html">Eclipse Magazin Vol. 2.2010</a>, 2010/01/27 [de]</li>
+  <li>Tests f&uuml;r die Tests by Marc R. Hoffmann, Eclipse Magazin Vol. 3.2011, 2011/03/25 [de]</li>
+  <li>EclEmma by Richard Attermeyer, Eclipse Magazin Vol. 2.2010, 2010/01/27 [de]</li>
   <li><a class="extern" href="http://agile.csc.ncsu.edu/SEMaterials/tutorials/eclemma/">Test Coverage with EclEmma</a> by Laurie Williams, Ben Smith and Sarah Heckman, <a class="extern" href="http://www.ncsu.edu/">North Carolina State University</a>, 2009/03/30</li>
-  <li>Testabdeckung mit Open-Source-Tools by Tim Wellhausen, <a class="extern" href="http://www.sigs-datacom.de/sd/publications/js/2008/06/index.htm">JavaSpektrum</a>, 2008/06/01 [de]</li>
```
Comment (marchof) [inline content/resources.html]: Also JavaSpektum is a printed magazine, so I prefer to keep the reference.

Comment (Godin) [inline content/resources.html]: @marchof fixed

**Comment by marchof** [comment]
@Godin Thanks!

**Pull request merged**